src/mongo_client.ts

```diff
@@ -248,6 +248,8 @@ export interface MongoClientOptions extends BSONSerializeOptions, SupportedNodeC
   autoEncryption?: AutoEncryptionOptions;
   /** Allows a wrapping driver to amend the client metadata generated by the driver to include information about the wrapping driver */
   driverInfo?: DriverInfo;
+  /** Configures a Socks5 proxy used for creating TCP connections. */
+  proxyOptions?: ProxyOptions;
```

Comment:
I think instead of a sub object we should have each `proxy*` option defined directly on the MongoClientOptions interface, since that's how the uri options work. IIUC

Reply:
ty!
@durran Just want to you to catch this thread as to why we're making the changes over in libmongocrypt